src/snowflake/cli/api/cli_global_context.py

```diff
@@ -213,6 +214,26 @@ def get_cli_context() -> _CliGlobalContextAccess:
     return _CliGlobalContextAccess(get_cli_context_manager())
 
 
+def start_cli_metrics_span(span_name: str):
```

Comment:
I don't think "start" makes sense here, since this both starts and stops the span at the appropriate time. Let's workshop the name of this + the `start_span` method it calls.

Reply:
+1, I feel @span would suffice

Reply:
Changed both the method name and the decorator name to just `span`, let me know if this works for you